semtypes/src/main/java/io/ballerina/types/typeops/IntBooleanPair.java

```diff
@@ -15,26 +15,27 @@
  *  specific language governing permissions and limitations
  *  under the License.
  */
-package io.ballerina.types;
+package io.ballerina.types.typeops;
```

Comment:
Shall we make this and IntListPair into generic `Pair` class or maybe `TwoTuple` class

Reply:
`FixedLengthArraySemtypePair` can also be replaced with this